frontend/src/components/UserPermissionHistoryTable/PermissionDetailsCol.vue

```diff
@@ -1,62 +1,77 @@
 <script setup lang="ts">
-import Chip from 'primevue/chip';
-import { formatForestClientDisplayName } from '@/utils/ForestClientUtils';
+import { PrivilegeDetailsPermissionTypeEnum } from 'fam-app-acsctl-api/model';
 import type { PrivilegeDetailsSchema } from 'fam-app-acsctl-api/model/privilege-details-schema';
+import Chip from 'primevue/chip';
+import OrganizationsList from '@/components/UserPermissionHistoryTable/OrganizationList.vue';
 
 const props = defineProps<{
     permissionDetails: PrivilegeDetailsSchema;
+    permissionChangeType: String;
 }>();
 
+enum PermissionChangeType {
+    GRANT = 'GRANT',
+    REVOKE = 'REVOKE'
+};
```

Comment:
Let's get client-code-gen to give us this value instead of defining here. It might not change often but it's a good idea to make the front-end somewhat independent 

We might need to redefine some schemas

![image](https://github.com/user-attachments/assets/dae51454-ae5f-4e3e-88f8-460d35ec4d85)

Reply:
No problem, initially I thought about the same thing too but didn't get to it at the backend, this is now adjusted. Thanks.